documentation/docs/tirith-policies/tirith-policy-error-tolerance.md

```diff
@@ -35,4 +35,27 @@ import TabItem from '@theme/TabItem';
       "item": "https://www.lambdatest.com/support/docs/tirith-policy-error-tolerance/"
     }]
   })
-}}></script>
\ No newline at end of file
+}}></script>
+Tirith supports error tolerance to handle missing or invalid keys in the input JSON.
+
+## Handling Missing Keys
+
+If a key referenced in an evaluator is missing, the evaluation typically fails. However, setting `error_tolerance` allows `evaluators` to be skipped instead of failing.
+
+If we modify the previous policy to include `error_tolerance`, missing keys won’t cause a failure:
+```javascript title="policy.json"
+{
```

Comment:
could you place a `...` in the top and the bottom to signify that this is an incomplete policy?